__tests__/fixtures/install/install-production-without-dev/yarn.lock

```diff
@@ -0,0 +1,1219 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
That is quite a big dependency graph, it would increase the test run time.
Is it possible to get cover the case with a smaller example?